scripts/image-builder/configure.sh

```diff
@@ -13,11 +13,28 @@ auth_file_path = "/etc/osbuild-worker/pull-secret.json"
 EOF
 fi
 
-sudo dnf install -y git osbuild-composer composer-cli ostree rpm-ostree \
+# osbuild from COPR to install version that:
+# 1. can build rhel-93 images
+# 2. doesn't have issues with unexpected mirror's RPM verification
+#    (like the one in 9.3 beta at the moment of writing this comment)
+sudo dnf copr enable -y @osbuild/osbuild
+sudo dnf copr enable -y @osbuild/osbuild-composer rhel-9-x86_64
+
+sudo dnf install -y \
```

Comment:
Can we do something to say "at least" this version, so that when it makes it into RHEL we get newer versions without having to change this code? I don't even know if that's possible with dnf.

Reply:
That change was part of https://github.com/openshift/microshift/pull/2503.
I think if we'd remove explicit, we'd always get newest one?
Just FYI: these are main branch builds, so bleeding edge ;)

Reply:
Ideally, we should stop using copr as soon as the functionality is integrated to RHEL. So, I would add a comment about this here, i.e. FIXME, etc.
For packages from copr, I recommended using something specific to avoid breaking our CI, etc.